### PR TITLE
WIP draft forcats 0.4.0 release

### PR DESCRIPTION
**Pull request by batpigandme**
Two points to resolve:

1. The new `fct_collapse()` `group_other` argument isn't working as expected right now/in 0.4.0. Should we exclude it as a result? https://github.com/tidyverse/forcats/issues/172

~2. I cannot figure out why the `as_factor()` bullet is indented under the preceding code-chunk. I've even checked the whitespace characters... If anyone has hints, I will be eternally grateful.~ (Still doing this locally, but seems right in the deploy preview).

### CODE REVIEW

**Comment by gtm19** [comment]
@batpigandme regarding the [`fct_collapse` issue](https://github.com/tidyverse/forcats/issues/172), you may want to check out pull request (#176) which fixes this issue, as well as reordering the factor levels so "other" is last, for consistency with other functions in the package.

**Comment by batpigandme** [comment]
@gtm19 Yes, I saw forcats/#176, and it looks good -- it's just a question of what to include in this post, since 0.4.0 is what's currently on CRAN, and that fix wasn't part of it (this is a very belated release post).

**Comment by gtm19** [comment]
@batpigandme gotcha. As you were.

**Comment by hadley** [comment]
Yeah, let's drop `fct_collapse()`

File: content/articles/2019-05-forcats-0-4-0.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+title: forcats 0.4.0
+author: Mara Averick
+date: '2019-05-10'
+slug: forcats-0-4-0
+description: > 
+  What's new in forcats 0.4.0?
+categories:
+  - package
+photo:
+  url: https://www.pexels.com/photo/close-up-photo-of-a-hand-holding-three-white-kittens-1643456/
```
Comment (jimhester) [inline content/articles/2019-05-forcats-0-4-0.Rmarkdown]: 🚨🚨🚨
There are only _three_ cats in this photo!!!
🚨🚨🚨

Comment (batpigandme) [reply content/articles/2019-05-forcats-0-4-0.Rmarkdown]: I know! 😂

Comment (hadley) [reply content/articles/2019-05-forcats-0-4-0.Rmarkdown]: The fourth cat is hiding behind the other 3

**Pull request merged**